staging/src/k8s.io/api/admission/v1beta1/types.go

```diff
@@ -94,6 +94,12 @@ type AdmissionResponse struct {
 	// The type of Patch. Currently we only allow "JSONPatch".
 	// +optional
 	PatchType *PatchType `json:"patchType,omitempty" protobuf:"bytes,5,opt,name=patchType"`
+
+	// Annotations is an unstructured key value map set by remote admission controller (e.g. error=image-blacklisted).
+	// MutatingAdmissionWebhook and ValidatingAdmissionWebhook admission controller will fill in the Annotations with
+	// fully qualified name (e.g. imagepolicy.example.com.mutatingwebhook.admissionregistration.k8s.io/error=image-blacklisted)
```

Comment:
I don't know about this key... maybe it should just be prefixed with the webhok name? I.e. `imagepolicy.example.com/error=image-blacklisted`

Reply:
webhook.Name can theoretically overlap with the built in names.

FYI: https://github.com/kubernetes/kubernetes/pull/58679#discussion_r165335854

What do you think?

Reply:
IMO we can just trust the admins to ensure that there aren't collisions. I think validating that the annotations are fo the expected form is the next best thing.
@sttts

Reply:
I could live without the postfix, i.e. what @tallclair suggests in the first comment.

Reply:
Done.